rust/src/tests/coordinator.rs

```diff
@@ -74,3 +74,89 @@ async fn full_cycle_1_round_1_participant() {
         }
     }
 }
+
+#[tokio::test]
+async fn dropout_1_participant_during_training() {
+    let settings = FederatedLearningSettings {
+        rounds: 1,
+        participants_ratio: 1.0,
+        min_clients: 2,
+        heartbeat_timeout: 5,
```

Comment:
We could probably use a much lower value. I guess it need to be an integer though, so maybe `1`?